### DISCUSSION TITLE
LNUG 95 - June 24 2020

### DISCUSSION THREAD

**Discussion by phazonoverload** [lnug/feedback#118] [General]
@lnug/organisers 

Thought I'd spin up this thread for event feedback on LNUG 95. 

1. Event worked well logistically - I think everyone who attended the networking turned on their mic and camera.
1. 28 people - a big reduction on LNUG 94. I wonder why this is? Maybe because it was a nice day out?
1. Timing was MUCH better than LNUG 94. Talks didn't run over, and the end wasn't rushed - it found it's natural conclusion. 
1. My team noted that, even though there were only two speakers, everyone who took the stage was a white man. As a note, we generally refrain from supporting events where this is the case. 

[Videos are here](https://drive.google.com/drive/folders/1aIJFG2T8ZAXADeF0tD2x1g93kgi4UUgQ?usp=sharing). I missed the very start of Aris' but that was only CityJS stuff. The main talk content is there.

Our general setup with events is we're happy to support three times - so if you want us to support LNUG July we're able to do that as long as the lineup is more diverse. After that I'm happy to help without my Vonage hat on - moving back to livestreaming + Social Cafe.

**Comment by admataz** [comment]
Hi @phazonoverload 
Thanks for this feedback - sorry, I took a break after the event and have just got back to my inbox. 

To address the points briefly:

1. I had lots of good feedback about the platform and that side of things. Thank you again for organising
2. We can only guess really why one month is better attended than another - I was disappointed at the turnout. A few things: LNUG 94 had a bigger draw speaker lineup - particularly with Anna being a node core contributor and well known to a broad community.  And yes, the weather. It happens every summer to an extent. And I think there's a degree of Zoom fatigue in the world. Either way, numbers are far lower than I would have expected and we need to think about our event in terms of promotion and content - and keep LNUG a relevant worthwhile event for our communities to attend virtually. 
3. Yes, we were better with time - two talks good, three talks - not so good.... and it helps that speakers kept to their times too. 
4. Yes, good  - I'm glad this has been raised and pointed out in these terms. It's a long standing problem with LNUG, and our attempts to address this have had limited lasting success so far. Some months are better than others, but there's no hiding that in general LNUG activity from organisers and speakers is largely characterised by white men, and it is something that needs change. I appreciate and value that you have a policy around this. 

Thanks for the videos. 

And thanks for the heads up about the three-times support limit from Vonage. I'm sure we will be likely to  accept support for 22 July and we can consider future arrangements/hosts/supporters/self-hosted between now and then.
